### PR TITLE
chore(package): update Web Components version to 1.x

### PR DESCRIPTION
**Pull request by asudoh**
### Description

Updates `version` field of Web Components' `package.json` to `1.x` as we release Web Components package as GA as of `v1.15.0`.

### Changelog

**Changed**

- The `version` field of `packages/web-components/package.json`.

### CODE REVIEW

**Comment by asudoh** [comment]
@jeffchew Do we need to update any release script for this? Thanks!

File: packages/web-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carbon/ibmdotcom-web-components",
-  "version": "0.8.0-rc.0",
+  "version": "1.15.0-rc.0",
```
Comment (jeffchew) [inline packages/web-components/package.json:3]: Thank you for updating @asudoh!

We should actually bump to `1.0.0-rc.0`, right?

Comment (asudoh) [reply packages/web-components/package.json:3]: Yeah wondering which was better, will update. Thanks @jeffchew for your feedback!

**Review by jeffchew — APPROVED** [review]
Thank you @asudoh !

**Pull request merged**